pf_driver/include/pf_driver/pf/pf_interface.h

```diff
@@ -30,8 +30,16 @@ class PFInterface
         }
         protocol_interface_ = std::make_shared<PFSDPBase>(ip_);
 
-        param_server_R2000_.setCallback(boost::bind(&PFInterface::reconfig_callback_r2000, this, boost::placeholders::_1, boost::placeholders::_2));
-        param_server_R2300_.setCallback(boost::bind(&PFInterface::reconfig_callback_r2300, this, boost::placeholders::_1, boost::placeholders::_2));
+        if(expected_device_ == "R2000")
```

Comment:
I like this idea, but maybe it more sense to do this here https://github.com/PepperlFuchs/ROS_driver/blob/64e3620341323e6fac48a3259e5de55feeb60a87/pf_driver/src/pf/pf_interface.cpp#L62-L67

Reply:
I think that generally makes sense, though I'm not 100% sure I'd expect a "handle_version" function do be setting up reconfigure callbacks.  I think I might make a new function call to put in `init()` right after the call to `handle_version`...